code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1280,3 +1313,21 @@
 
 /obj/item/bodypart/proc/un_paralyze()
 	REMOVE_TRAITS_IN(src, EMP_TRAIT)
+
+/// Returns the generic description of our BIO_EXTERNAL feature(s), prioritizing certain ones over others. Returns error on failure.
+/obj/item/bodypart/proc/get_external_description()
+	if (biological_state & BIO_FLESH)
+		return "flesh"
+	if (biological_state & BIO_WIRED)
```

Comment:
This is probably very late into the review to pick up on this, but does it make sense for robotic limbs to have wiring on the outside, and the metal plating on the interior?

Reply:
its not really about it making sense, its mostly about it working the same as flesh/bone

if wiring was internal, things would get a bit funky due to how slash/pierce is translated to blunt when external is damaged